README.md

```diff
@@ -7,9 +7,10 @@ A collection of scripts for common data management and processing tasks
 | ----------- | -------- | ----------- |
 | Demux a standard sequencing run | `evros demux.bcl2fastq --exp_id YYMMDD_EXP_ID` | Assumes your sample sheet is uploaded to S3.|
 | Demux a 10X run | `evros demux.10x_mkfastq --exp_id YYMMDD_EXP_ID` | Again, assumes a sample sheet is present on S3 |
-| Align a 10X run | ```evros alignment.10x_count --taxon [see --help for options] --s3_input_dir s3://input-bucket/path/to/fastqs --s3_output_dir s3://output-bucket/path/for/results``` | Run once for each channel of the run. Very slow! |
-| Align using STAR and htseq (slow) | ```evros alignment.run_star_and_htseq --taxon [see --help for options] --num_partitions NUM_PARTITIONS --partition_id PARTITION_ID --s3_input_path s3://input-bucket/path/to/fastqs --s3_output_path s3://output-bucket/path/for/results``` | Run once for each channel of the run. Very slow! |
-| Align using STAR and htseq (preferred) | ```aws_star [see --help for more information] taxon num_partitions s3://input-bucket/path/to/fastqs --s3_output_path s3://output-bucket/path/for/results > your_script.sh``` | Creates a shell script locally to launch many alignments using `source your_script.sh` |
+| Align one sample using 10X | ```evros alignment.run_10x_count [see --help for input format] --taxon [see --help for options] --s3_input_path s3://input-bucket/path/to/fastqs --s3_output_path s3://output-bucket/path/for/results``` | Run once for each channel of the run. Suitable for aligning only one sample. Very slow for multiple samples! |
+| Align multiple samples using 10X | ```aws_10x [see --help for input format] --taxon TAXON --num_partitions NUM_PARTITIONS --s3_input_path s3://input-bucket/path/to/fastqs --s3_output_path s3://output-bucket/path/for/results > your_script.sh``` | Creates a shell script locally to launch many alignments using `source your_script.sh` |
+| Align one sample using STAR and htseq | ```evros alignment.run_star_and_htseq [see --help for input format] --taxon [see --help for options] --num_partitions NUM_PARTITIONS --partition_id PARTITION_ID --s3_input_path s3://input-bucket/path/to/fastqs --s3_output_path s3://output-bucket/path/for/results``` | Run once for each channel of the run. Suitable for aligning only one sample. Very slow for multiple samples! |
```

Comment:
I wouldn't say this is only for one sample, it's really for running 10-20 at a time. But useful to note that it's not intended for a whole run.